lib/puppet/indirector/certificate_revocation_list/rest.rb

```diff
@@ -7,4 +7,16 @@ class Puppet::SSL::CertificateRevocationList::Rest < Puppet::Indirector::REST
   use_server_setting(:ca_server)
   use_port_setting(:ca_port)
   use_srv_service(:ca)
+
+  def find(request)
```

Comment:
Would we ever execute a find via the `file` terminus and run into the same problem?